src/connection.js

```diff
@@ -30,7 +30,10 @@ class FtpConnection extends EventEmitter {
       this.server.emit('client-error', {connection: this, context: 'commandSocket', error: err});
     });
     this.commandSocket.on('data', this._handleData.bind(this));
-    this.commandSocket.on('timeout', () => {});
+    this.commandSocket.on('timeout', () => {
+      this.log.error(err, 'Client timeout');
+      this.close();
```

Comment:
If `close` rejects it will be an unhandled rejection.

Can we add a `.catch` with a trace log?

```js
this.close()
.catch(err => this.log.trace(err, 'Client close error'));
```

Reply:
Got it
Thank you!